### PR TITLE
Feature: Fetch profile data

### PR DESCRIPTION
**Pull request by at-sonvu**

### CODE REVIEW

File: MyApp/View/Controllers/Login/LoginViewController.swift [hunk 1]
```diff
@@ -12,15 +12,42 @@ class LoginViewController: ViewController {
 
     @IBOutlet weak var emailTextField: UITextField!
     @IBOutlet weak var passwordTextField: UITextField!
+
+    // MARK: - Properties
     var viewModel = LoginViewModel(user: nil)
 
-    func textField(for field: LoginViewModel.Field) -> UITextField {
```
Comment (at-daiho) [inline MyApp/View/Controllers/Login/LoginViewController.swift]: WARNING - Limit vertical whitespace to a single empty line. Currently 2.

**Comment by at-daiho** [comment]
Total linter violations in pull request: 1
Serious: 0
Warnings: 1

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: MyApp/Model/Schema/Profile.swift [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    dynamic var balance = 0
+    dynamic var imagePath = ""
+
+    override static func primaryKey() -> String? {
+        return "id"
+    }
+
+    func mapping(map: Map) {
+        email <- map["authId"]
+        imagePath <- map["userProfile.avatar"]
```
Comment (at-tuanmai2) [inline MyApp/Model/Schema/Profile.swift]: imagePath?

**Pull request merged**